### PR TITLE
Update to webpack 3 and all related dependencies

### PR DESCRIPTION
**Pull request by jhnns**
This PR updates webpack and all related dependencies to the latest version on NPM, particularly:

- webpack
- css-loader
- sass-loader
- node-sass
- style-loader
- file-loader
- extract-text-webpack-plugin
- babel-loader
- svg-sprite-loader
- webpack-dev-server

I also decided to replace the forked storybook project from `@kadira/storybook` with the original one again since it seems like the forked project is not actively maintained anymore (I couldn't find a repository for it). This also involved switching from `@kadira/react-storybook-decorator-centered` to `@storybook/addon-centered` which is the officially maintained verison of that dependency now. In order to fix deprecation warnings, I also added `@storybook/addon-actions` which replaces the builtin actions (but hasn't been changed otherwise).

All the tests are running and a smoke test on the frontend and the storybook looked good so far. It is possible that HMR is not working as expected, but as far as I understood @webholics, it wasn't reliable anyway. I can take a look if that is an issue, but I'd try to merge this one first in order to update the outdated dependencies.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "build:styleguide": "build-storybook -c .storybook -o dist/styleguide",
     "start:dev": "PORT=3000 NODE_ENV=development babel-node bin/app-dev-server",
     "start:dev:stubs": "PORT=3000 NODE_ENV=development-with-stubs babel-node bin/app-dev-server",
-    "start:styleguide": "start-storybook --dont-track -p 3001 -s src",
```
Comment (jhnns) [inline package.json]: That argument has been removed. DNT is the default now.

**Comment by jhnns** [comment]
Looks like there's an issue with svg-sprite-loader... I'm on it.

**Comment by jhnns** [comment]
I had to change the Icon component interface because the svg-sprite-loader is now returning an object with an id.

During refactor, I recognized that it would be handy if we could have a unit test that shallow renders each story in node and checks for react proptype errors. Otherwise, you have to click through every component in the storybook. I've created an issue for that to discuss it #320

The PR is ready to be reviewed now

**Comment by webholics** [comment]
Thx for the explanations! Good job!

**Pull request merged**